From 6aa4c96481ac376cbf83f35c807521092402568e Mon Sep 17 00:00:00 2001
From: Vlad Zahorodnii <vlad.zahorodnii@kde.org>
Date: Wed, 12 Mar 2025 07:50:30 +0000
Subject: [PATCH] kcms/rules: Remove pragma on OptionsComboBox

It is suggested by qmlls, but this seems to cause graphical artifacts
on some systems, rendering the comboboxes unusable

BUG: 501357


(cherry picked from commit ee2ebfee4ce5bf2deda9a8c914902b49321cd266)

Co-authored-by: Ismael Asensio <isma.af@gmail.com>
---
 src/kcms/rules/ui/OptionsComboBox.qml | 2 --
 1 file changed, 2 deletions(-)

diff --git a/src/kcms/rules/ui/OptionsComboBox.qml b/src/kcms/rules/ui/OptionsComboBox.qml
index 5a5d02eee90..d8a2eb8bf09 100644
--- a/src/kcms/rules/ui/OptionsComboBox.qml
+++ b/src/kcms/rules/ui/OptionsComboBox.qml
@@ -11,8 +11,6 @@ import QtQuick.Controls as QQC2
 import org.kde.kirigami 2.10 as Kirigami
 import org.kde.kcms.kwinrules
 
-pragma ComponentBehavior: Bound
-
 
 QQC2.ComboBox {
     id: optionsCombo
-- 
GitLab

